lib/openai/base.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative "../http/http"
+
+module Openai
+  # Base class of openai
+  class Base
+    @openai_api_key = nil
```

Comment:
you don't need to predefine like this, I think it's nil by default

Reply:
thanks anh for the infor, I removed it!